backend/src/test/java/com/redhat/cloud/notifications/recipients/rbac/AuthRequestFilterTest.java

```diff
@@ -13,36 +16,73 @@
 import java.util.Base64;
 import java.util.List;
 
+@QuarkusTest
 public class AuthRequestFilterTest {
 
+    private static final String testToken = "{\"approval\":{\"secret\":\"123\"},\"advisor\":{\"secret\":\"456\"},\"notifications\":{\"secret\":\"789\"}}";
+
+    @Inject
+    ObjectMapper objectMapper;
+
     @BeforeEach
     public void clean() {
-        System.setProperty("rbac.service-to-service.exceptional.auth.info", "");
+        System.clearProperty(AuthRequestFilter.RBAC_SERVICE_TO_SERVICE_DEV_EXCEPTIONAL_AUTH_KEY);
+        System.clearProperty(AuthRequestFilter.RBAC_SERVICE_TO_SERVICE_APPLICATION_KEY);
+        System.clearProperty(AuthRequestFilter.RBAC_SERVICE_TO_SERVICE_SECRET_MAP_KEY);
+    }
+
+    private AuthRequestFilter getAuthRequestFilter() {
+        // Injecting doesn't seem to take into account the System.setProperty calls.
+        // Probably related to: https://quarkus.io/blog/quarkus-test-profiles/
```

Comment:
That's probably because the filter is `@ApplicationScoped` and the configuration is loaded during the `@PostConstruct` phase. All tests use the same filter instance when it's injected.